mu-plugins/blocks/global-header-footer/src/footer/index.js

```diff
@@ -13,9 +13,10 @@ import { useBlockProps } from '@wordpress/block-editor';
 import metadata from './block.json';
 
 const variations = [
-	{ label: __( 'White on black (default)', 'wporg' ), value: 'white-on-black' },
+	{ label: __( 'White on charcoal-1', 'wporg' ), value: 'white-on-dark-black' },
```

Comment:
Nothing is using this value yet, correct? So it's not too late to try to improve the naming? For me the range of values is where it's going to get messy if/when we add more variations...

```suggestion
	{ label: __( 'White on charcoal-1', 'wporg' ), value: 'white-on-charcoal-1' },
```

Also I wonder if it's still manageable to migrate the existing usages to new values while we are still relatively early in the overall redesign. All the pages are saved as patterns in source control in the various repos, so we could replace eg. `white-on-black` with `white-on-charcoal-2`, and deploy all at once.

Or maybe I'm worrying too much about this... feel free to give me a reality check 😂

Reply:
For `white-on-black`
1 usage in wporg-main-2022
2 usages in wporg-showcase-2022
2 usages in wporg-parent-2021
Several in this repo

Reply:
Usages are even better for `white-on-blue`

Reply:
@adamwoodnz Perhaps for a migration strategy we could do the following:
1. Add both the legacy and new values to the [switch statement](https://github.com/WordPress/wporg-mu-plugins/pull/307/files#diff-460e50b82495cab49b3ce3a702572bec919fae2cbec691937f8abbe7de6e2c33R1031) so we can support both the legacy and new new names.
2. Create an issue so we can update the content someday.
3. Eventually remove the support from the switch statements.

What do you think? I'm going to hold off until we find out whether we are going to use charcoal-1 or charcoal-2, but I think it's a good idea to implement someday.

Reply:
Yeah I was a bit hesitant to suggest we reuse the same color. I think that makes sense for simplicity's sake.

Yeah having both values in the switch could work.

Reply:
Would a [deprecation](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) work? It might only apply to JS blocks.

If so, I think the proposed migration strategy sounds good.